scspell/__init__.py

```diff
@@ -965,5 +995,6 @@ def main():
                            args.relative_to,
                            args.report,
                            args.c_escapes,
-                           args.test_input)
-        return 0 if okay else 1
+                           args.test_input,
+                           args.list_unique)
+        return 0 if okay else 1
```

Comment:
you remove the EOL here.

maybe you need to use a different editor to re-add this.
`vi` definitely lets you control this.

there is a vim for windows, if you are so afflicted.

then use `git commit --amend -a` to re-create this commit , and the git push , with `--force` to overwrite this commit.